app/src/main/java/com/it3048/mobiledeviceproject/MainActivity.kt

```diff
@@ -41,15 +32,15 @@ class MainActivity : AppCompatActivity() {
         setSupportActionBar(findViewById(R.id.toolbar))
 
         /** Set up list */
-        meetingList = ArrayList()
+        meetingList = ArrayList<Meeting>()
         recyclerView = findViewById(R.id.eventsList)
         meetingAdapter = MeetingAdapter(this, meetingList)
         recyclerView.adapter = meetingAdapter
         val title = meetingTitle.text.toString()
         val date = meetingDate.text.toString()
         val description = meetingDescription.text.toString()
         val link = meetingLink.text.toString()
-        meetingList.add(MeetingDAO("$title", "$date", "$description", "Location: $link"))
+        meetingList.add(Meeting(0, title, date, description, "Location: $link"))
```

Comment:
Utilized the DTO class instead of a "DAO" class to fulfill the purpose of having a DTO class. As DTOs are data objects, they should be used instead of creating another class to do so. This will also make using your Room DB much easier.

Sources: https://martinfowler.com/eaaCatalog/dataTransferObject.html
https://stackoverflow.com/questions/1051182/what-is-a-data-transfer-object-dto

Reply:
Good.